OpenRA.Game/Sync.cs

```diff
@@ -27,6 +27,8 @@ public interface ISync { }
 
 	public static class Sync
 	{
+		static DebugSettings debugSettings = new DebugSettings();
```

Comment:
This will never be used before `Initialize` was called right?
```suggestion
		static DebugSettings debugSettings;
```